src/test/resources/application.properties

```diff
@@ -51,3 +51,5 @@ odoo.database=odoo-db
 odoo.username=test@odoo.test
 odoo.password=test
 odoo.customer.address.type=contact
+
+should.sync.any.patient.asCustomer=false
```

Comment:
Please also add the property to the root application.properties file under the config folder with proper inline documentation

Reply:
Yes @wluyima indeed @wluyima. It was added [here](https://github.com/mekomsolutions/eip-routes-odoo/pull/2/files#diff-ef406ecd699d5c3b08b2c5f7336d701a171bac4069de7bee907b6c3cd27386f8R37-R38).